wolox-books/src/services/localStorageService.js

```diff
@@ -12,3 +12,11 @@ export function removeValue(key) {
   const encodedKey = window.btoa(`@@AUTH:${key}`)
   window.localStorage.removeItem(encodedKey)
 }
+
+export function getValue(key) {
+  const encodedKey = window.btoa(`@@AUTH:${key}`)
+  let encodedValue
+  encodedValue = window.localStorage.getItem(encodedKey)
```

Comment:
Merge this two lines, no need to create the value and in the other line set the value